functional/FunctionalTest.java

```diff
@@ -2473,12 +2478,23 @@ public static void main(String[] args) {
       client = new MinioClient(endpoint, accessKey, secretKey);
       // Enable trace for debugging.
       // client.traceOn(System.out);
-      FunctionalTest.runTests();
-
-      // Run fast test with region parameter passed to the constructor
-      client = new MinioClient(endpoint, accessKey, secretKey, region);
-      FunctionalTest.runFastTests();
 
+      // For mint environment, run tests based on mint mode
+      if (mintEnv) {
```

Comment:
This would mean that functional tests won't be run in full by default.. I thought its the other way around if needed for quick tests MINT_MODE is set to `quick` if not we run full mode for both mint and regular SDK tests.  

This also simplifies the condition we have below.

Reply:
No.  Accordingly to mint, mint's default mode is `quick`

Reply:
It won't be correct for quick to be default, the tests are not sufficient to validate the server.  Most of us will happily think server is working fine.  MINT_MODE=quick is only needed on special situations i.e CI/CD envs . So i don't see why special situations should be default behavior.

Reply:
Understood after discussing with @balamurugana ,we have to properly pick the tests for sanity. Right now minio-java has chosen enough set of tests for sanity (there is room to add more). But this is what we need to repeat across all our libraries.